tests/integration/test_registry.py

```diff
@@ -10,13 +10,7 @@
     "kubeinterface_name, kuberegistry_name",
     [
         ("kubeinterface", "kube_registry"),
-        pytest.param(
-            "lightkubeinterface",
-            "lightkube_registry",
-            marks=pytest.mark.xfail(
-                reason="https://warthogs.atlassian.net/browse/DPE-2131"
-            ),
-        ),
+        ("lightkubeinterface", "lightkube_registry"),
```

Comment:
**praise:** Thank god, I almost left a comment on the nesting being really strange here in the other PR!

Reply:
Not my fault, it was `black`.... :-)